CreeDictionary/CreeDictionary/relabelling.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import annotations
+
+import csv
+from enum import IntEnum
+from typing import Iterable, Optional, TextIO, Tuple, TypeVar, Union
+
+from utils import shared_res_dir
+from utils.types import FSTTag, Label
+
+CRK_ALTERNATE_LABELS_FILE = shared_res_dir / "crk.altlabel.tsv"
+
+Default = TypeVar("Default")
+
+
+class _LabelFriendliness(IntEnum):
+    """
+    Weird enum that I'm not sure should have ever existed.
```

Comment:
- `SOURCE_LANGUAGE` and `TARGET_LANGUAGE` would be more language-independent
  - You can always parse the header to get the column names and let them flow through to the HTML templates; that is, if someone adds a “foo” column in the .tsv, that would then be accessible as `{{ label.foo }}`. But not having looked at how exactly these are used, that may be unhelpfully abstract and a distraction from your present refactoring goals.

Reply:
Yeah, this is hard-coded to the structure of `crk.altlabel.tsv`. I won't fix it in this PR, because this PR is me getting things ready for paradigm relabelling, but it's a thing to consider when making this language-independent.

Reply:
But this can be more complex than just source and target language. In the case of itwêwina, you would have both linguistic and plain/non-linguistic forms in the target language, and plain/non-linguistic forms in the source language. Also, the linguistic labels can be short or long.

To keep things simple, perhaps one would have a two-way division under the source and target languages, into linguistic and non-linguistic (or plain), but how these are named and accessible in the interface may need to be specified per each language.

In the case of Cree, these relabeling schmemes would be:
1a. Target, linguistic
1b. Target, plain (English)
2b. Target, (plain) nêhiyawêwin